supertokens_python/recipe/thirdpartyemailpassword/interfaces.py

```diff
@@ -29,6 +15,28 @@
 ThirdPartyAPIOptions = ThirdPartyInterfaces.APIOptions
 EmailPasswordAPIOptions = EPInterfaces.APIOptions
 
+# Exporting re-used classes
+CreateResetPasswordOkResult = EPInterfaces.CreateResetPasswordOkResult
+CreateResetPasswordWrongUserIdErrorResult = EPInterfaces.CreateResetPasswordWrongUserIdErrorResult
+EmailExistsGetOkResponse = EPInterfaces.EmailExistsGetOkResponse
+GeneratePasswordResetTokenPostOkResponse = EPInterfaces.GeneratePasswordResetTokenPostOkResponse
+PasswordResetPostInvalidTokenResponse = EPInterfaces.PasswordResetPostInvalidTokenResponse
+PasswordResetPostOkResponse = EPInterfaces.PasswordResetPostOkResponse
+ResetPasswordUsingTokenInvalidTokenErrorResult = EPInterfaces.ResetPasswordUsingTokenInvalidTokenErrorResult
+ResetPasswordUsingTokenOkResult = EPInterfaces.ResetPasswordUsingTokenOkResult
+SignInPostWrongCredentialsErrorResponse = EPInterfaces.SignInPostWrongCredentialsErrorResponse
+SignInWrongCredentialsErrorResult = EPInterfaces.SignInWrongCredentialsErrorResult
+SignUpEmailAlreadyExistsErrorResult = EPInterfaces.SignUpEmailAlreadyExistsErrorResult
+SignUpPostEmailAlreadyExistsErrorResponse = EPInterfaces.SignUpPostEmailAlreadyExistsErrorResponse
+UpdateEmailOrPasswordEmailAlreadyExistsErrorResult = EPInterfaces.UpdateEmailOrPasswordEmailAlreadyExistsErrorResult
+UpdateEmailOrPasswordOkResult = EPInterfaces.UpdateEmailOrPasswordOkResult
+UpdateEmailOrPasswordUnknownUserIdErrorResult = EPInterfaces.UpdateEmailOrPasswordUnknownUserIdErrorResult
+
+AuthorisationUrlGetOkResponse = ThirdPartyInterfaces.AuthorisationUrlGetOkResponse
+SignInUpFieldErrorResult = ThirdPartyInterfaces.SignInUpFieldErrorResult
+SignInUpPostNoEmailGivenByProviderResponse = ThirdPartyInterfaces.SignInUpPostNoEmailGivenByProviderResponse
+SignInUpPostFieldErrorResponse = ThirdPartyInterfaces.SignInUpPostFieldErrorResponse
```

Comment:
Should be prefixed with ThirdParty since they are returned by a function that also has that prefix